spec/package.json

```diff
@@ -44,6 +44,7 @@
   "resolutions": {
     "nan": "file:../../third_party/nan",
     "dbus-native/optimist/minimist": "1.2.7",
-    "dbus-native/xml2js": "0.5.0"
+    "dbus-native/xml2js": "0.5.0",
+    "abstract-socket": "github:deepak1556/node-abstractsocket#121d94ca6d1811f90cd035e9787aaea79b5cf533"
```

Comment:
tldr for reviewers: moves from nan to napi

https://github.com/deepak1556/node-abstractsocket/commit/121d94ca6d1811f90cd035e9787aaea79b5cf533

Reply:
Given the module was used for test only, this was the faster route to unblock the roll also avoids any future breakage :)